bindings/java/pom.xml

```diff
@@ -32,7 +32,7 @@
 
     <groupId>org.apache.opendal</groupId>
     <artifactId>opendal-java</artifactId>
-    <version>0.2.0-SNAPSHOT</version>
+    <version>0.36.0-SNAPSHOT</version>
```

Comment:
@Xuanwo could you share the script we do this simultaneous updates? I'd like to ensure and investigate how to do Maven releases automatically.

Reply:
Thi is changed by hand,

Reply:
@Xuanwo I check the version on 0.36.0 tag and it's still `0.36.0-SNAPSHOT`.

For Java library releases, a formal release doesn't have a -SNAPSHOT suffix while a develop version does. Could you please update it like `0.36.0` for the following tags and keep it like `0.37.0-SNAPSHOT` on the main branch after tagging.

Reply:
Ok, we will change it to 0.37.0 in next release. And is snapshot version required? I prefer not to provide snapshot version if possible.

Reply:
That's OK for no snapshots. Please ping me when you call 0.37.0 release so that I can update the release process correspondingly.